zcash_primitives/CHANGELOG.md

```diff
@@ -10,6 +10,51 @@ and this library adheres to Rust's notion of
 ### Changed
 - Bumped dependencies to `bls12_381 0.8`, `ff 0.13`, `group 0.13`,
   `jubjub 0.10`, `orchard 0.4`, `sha2 0.10`, `bip0039 0.10`.
+- The type of the `sapling::SAPLING_COMMITMENT_TREE_DEPTH` constant has been
+  changed from `usize` to `u8`.
+
+### Moved
+- The following constants and methods have been moved from the
+  `merkle_tree::incremental` module into the `merkle_tree` module to
+  consolidate the serialization code for commitment tree frontiers:
+  - `SER_V1`
+  - `SER_V2`
```

Comment:
These are ultimately removed, so mentioning that they were "moved" in the changelog is perhaps superfluous.